client/backup_storage_location.go

```diff
@@ -41,6 +41,69 @@ func (h *V1Client) GetS3BackupStorageLocation(uid string) (*models.V1UserAssetsL
 	return resp.Payload, nil
 }
 
+// GetMinioBackupStorageLocation retrieves an existing Minio backup storage location.
+func (h *V1Client) GetMinioBackupStorageLocation(uid string) (*models.V1UserAssetsLocationS3, error) {
+	params := clientv1.NewV1UsersAssetsLocationMinioGetParamsWithContext(h.ctx).
+		WithUID(uid)
+	resp, err := h.Client.V1UsersAssetsLocationMinioGet(params)
+	if err != nil {
+		return nil, err
+	}
+	return resp.Payload, nil
+}
+
+// GetGCPBackupStorageLocation retrieves an existing GCP backup storage location.
+func (h *V1Client) GetGCPBackupStorageLocation(uid string) (*models.V1UserAssetsLocationGcp, error) {
+	params := clientv1.NewV1UsersAssetsLocationGcpGetParamsWithContext(h.ctx).
+		WithUID(uid)
+	resp, err := h.Client.V1UsersAssetsLocationGcpGet(params)
+	if err != nil {
+		return nil, err
+	}
+	return resp.Payload, nil
+}
+
+// GetAzureBackupStorageLocation retrieves an existing Azure backup storage location.
+func (h *V1Client) GetAzureBackupStorageLocation(uid string) (*models.V1UserAssetsLocationAzure, error) {
+	params := clientv1.NewV1UsersAssetsLocationAzureGetParamsWithContext(h.ctx).
+		WithUID(uid)
+	resp, err := h.Client.V1UsersAssetsLocationAzureGet(params)
+	if err != nil {
+		return nil, err
+	}
+	return resp.Payload, nil
+}
+
+// ValidateS3BackupStorageLocation validate a S3 credential for backup storage location.
+func (h *V1Client) ValidateS3BackupStorageLocation(awsCred *models.V1AwsS3BucketCredentials) error {
+	params := clientv1.NewV1AwsS3ValidateParamsWithContext(h.ctx).WithAwsS3Credential(awsCred)
+	_, err := h.Client.V1AwsS3Validate(params)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// ValidateGcpBackupStorageLocation validate a gcp credential for backup storage location.
+func (h *V1Client) ValidateGcpBackupStorageLocation(gcpCred *models.V1GcpAccountNameValidateSpec) error {
+	params := clientv1.NewV1GcpBucketNameValidateParamsWithContext(h.ctx).WithBody(gcpCred)
+	_, err := h.Client.V1GcpBucketNameValidate(params)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// ValidateAzureBackupStorageLocation validate an azure credential for backup storage location.
+func (h *V1Client) ValidateAzureBackupStorageLocation(azureCred *models.V1AzureCloudAccount) error {
+	params := clientv1.NewV1AzureAccountValidateParamsWithContext(h.ctx).WithAzureCloudAccount(azureCred)
+	_, err := h.Client.V1AzureAccountValidate(params)
+	if err != nil {
```

Comment:
please return the error directly for these 3 validate funcs

Reply:
yes by mistake, updated